HIP_scientific_research.md

```diff
@@ -0,0 +1,121 @@
+# HIP allocating unused processing power for scientific research.
+
+- Author(s): snipeTR
+- Start Date: 23.01.2023
+- Category: technical
+- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
+- Tracking Issue: <!-- leave this empty; maintainer will create a discussion issue -->
+
+# Summary
+
+The incredible amount of total processing power created by changing hotspot tasks with HIP 55 will enable changes to be used for one/multiple projects that will benefit the world.
+<!-- Read the content requests in all sections before starting to write any section. -->
+
+# Motivation
+
+With HIP 55, many HOTSPOT devices have turned into a type of hotspot called "light" hotspot, and the processing power used for block processing has become almost completely useless.
+
+ Currently, thousands of devices with extremely powerful processors and Terabytes of memory are sitting idle on roofs and in houses. These devices are always open to electricity and internet access, and many have advanced cooling systems. Because they are not performing any block processing, they are wasting energy and running idle until the next ping. 
```

Comment:
If we assume that the average hotspots have 4GB of memory and 984,195 hotspots are online at %48 each, it would mean that there are 472,413 active working hotspots. If we also assume that each one has a 1GHz, 2-core processor, the table would look like this: a total of 1,889,652 GB of total memory, which makes up 1,845.3 TB of total memory. Even if the real RAM is 1 GB on average, it would still have a potential memory of 461 TB. The CPU is calculated at 2 cores at 1GHz, which would mean 944,826 GHz of speed.